Sakura/Chat/response.py

```
@@ -0,0 +1,32 @@
+from typing import Optional, Dict
+
+from Sakura.Chat.chat import get_response
+from Sakura.Core.helpers import get_error, log_action
+from Sakura.Database.conversation import add_history
+
+
+async def get_response(
```

Ruff (0.13.3)

Redefinition of unused `get_response` from line 3

(F811)